components/organisms/RepositoriesCart/repositories-cart.tsx

```diff
@@ -54,64 +55,69 @@ const RepositoriesCart = ({
   };
 
   return (
-    <div className="w-full lg:w-[364px] border flex flex-col gap-2 rounded-lg p-6">
-      {/* Empty state of Cart */}
-      {cartItems.length > 0 && hasItems ? (
-        cartItems.map((item, index) => (
-          <div className="flex flex-col w-full" key={`${index}/${Math.random()}`}>
-            {item}
+    <>
+      <Title className="!text-1xl !leading-none mb-4 xl:mt-10 font-medium" level={4}>
+       Repositories Added
+      </Title>
+      <div className="w-full lg:w-[364px] border flex flex-col gap-2 rounded-lg p-6">
```

Comment:
```suggestion
      <div className="w-full lg:w-[22.75rem] border flex flex-col gap-2 rounded-lg p-6">
```

Reply:
going to move this conversation over here https://github.com/open-sauced/insights/issues/1522